src/css/TrackView.css

```diff
@@ -26,20 +31,53 @@
 }
 
 .errorIndicator {
-    background-color: lightcoral;
+    background-color: lightcoral!important; /* but I think it is a bad practice to use !important */

```

Comment:
it's better to avoid !important but sometimes it's needed.
in your case errorIndicator style most likely conflicting with trackContainer style. to fix this issue you can set errorIndicator style on child element of a trackContainer so there will be less specify collisions: https://www.w3.org/TR/selectors-3/#specificity

Reply:
It is very strange. I just removed the `"!important"` and it still works. Before it doesn't work withot `!important`.